src/Scripting/VisualBasicTest/BasicScriptingTest.vbproj

```diff
@@ -8,8 +8,8 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <OutputType>Library</OutputType>
     <AssemblyName>Microsoft.CodeAnalysis.VisualBasic.Scripting.UnitTests</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
-    <RoslynProjectType>UnitTestDesktop</RoslynProjectType>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
It never ran one CoreClr that I could find. I chatted with @tmat about this and the approach we decided is 

1. Just use net46 for now since that's what we're actually doing
1. Separately switch to multi-target and get it working